openstack/cbr/v3/tags/requests.go

```diff
@@ -0,0 +1,181 @@
+package tags
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+)
+
+/*
+ShowVaultProjectTag
+Query the set of all tags of the tenant in the specified Region and instance type
+
+@author Aloento
+@since 0.5.17
+@version 0.1.0
+*/
```

Comment:
@gtema should we keep this lines or better remove? We never added smth like this before.